processing/data_collection/gazette/spiders/ce_fortaleza.py

```diff
@@ -0,0 +1,55 @@
+import re
+import dateparser
+import w3lib.url
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+
+
+class CeFortalezaSpider(Spider):
+    GAZETTE_ELEMENT_CSS = '.diarios-oficiais .table-responsive tbody tr'
+    DATE_CSS = 'td:nth-child(2)::text'
+    EXTRA_CSS = 'td:nth-child(1)::text'
+    NEXT_PAGE_CSS = 'ul.pagination .page-link::attr(href)'
+
+    MUNICIPALITY_ID = '2304400'
+
+    allowed_domains = ['apps.fortaleza.ce.gov.br']
+    name = 'ce_fortaleza'
+    start_urls = ['http://apps.fortaleza.ce.gov.br/diariooficial/?mes-diario=todos']
+
+    def start_requests(self):
+        requests = []
+
+        for year in range(2015, datetime.now().year + 1):
```

Comment:
Date filtering is done in `GazetteDateFilteringPipeline` (https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/pipelines.py), so I think there is no problem in getting all gazettes available (no matter the year).

Reply:
Hello @rennerocha 
I made the other changes.

About this comment, the gazette of Fortaleza has files of 1952,  but the server is to much unstable to download all data. 
I should get all data or I keep filtering the year? if Yes, what range is more interesting?